Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -1678,6 +1845,27 @@ private HandleType GetHandleType(Transform handle)
             return HandleType.None;
         }
 
+        private bool TryGetPointerPoint(Handedness handed, out Vector3 point)
```

Comment:
Will this work for other input types, for example sixdof controllers? It is directly accessing the hand positions, not the pointer data.

Reply:
ahh some more properties have to be made public.

Reply:
ok ive pushed an update to the PR that sets the default values in the event that there is no corner prefab set. THis way, the defaults in the script are set for the shell parity handle prefabs, and if none are specified (the old style cube and sphere handles) the settings get set on Start(). If the BoundingBoxRuntimeExample scene is retried- the blue boxes start up and appear as illustrated above.